cmd/query/app/flags.go

```diff
@@ -100,8 +101,11 @@ func AddFlags(flagSet *flag.FlagSet) {
 func (qOpts *QueryOptions) InitFromViper(v *viper.Viper, logger *zap.Logger) (*QueryOptions, error) {
 	qOpts.HTTP.Endpoint = v.GetString(queryHTTPHostPort)
 	qOpts.GRPC.NetAddr.Endpoint = v.GetString(queryGRPCHostPort)
+	qOpts.GRPC.NetAddr.Transport = confignet.TransportTypeTCP
```

Comment:
couple of alternatives here: 
- define a `DefaultQueryOptions` here that sets `qOpts` to that configuration before overriding the flags
- expose an `ApplyDefaults` for `QueryOptions` that is used by all in one

Reply:
@yurishkuro went with the first approach here - let me know if you prefer something different here though.